pytorch_toolkit/ote/ote/modules/builder.py

```diff
@@ -40,8 +41,8 @@ def build_arg_parser(obj_type):
     return build(obj_type, ARG_PARSERS)
 
 
-def build_arg_converter_map(obj_type):
-    return build(obj_type, ARG_CONVERTER_MAPS)
+def build_arg_converter(obj_type):
```

Comment:
Please, Ilya, rename this function to `create_arg_converter`

The reason is uniformity of the code: it is unconvenient, when all `build_...` functions make just "taking from Registry", whereas one build function makes another operation.
I suppose that this break of uniformity should be reflected in the name of the function.

2 @evgeny-izutov:   Please, write your opinion?

Reply:
Actually the rule `all build_... functions make just "taking from Registry"` is not defined anywhere, so there is no reason to think in such way in the future. Moreover renaming breaks logical structure of such peace of code: some config info in the input and a functor in the output.

Reply:
So, it will not be renamed, I resolves the comment.

Reply:
Ok, as you wish. But my "minority report": I think that this change will decrease the quality of the code a bit.